tests/functional-ui/api.yaml

```diff
@@ -2,15 +2,10 @@ Log:
   Level: info
 
 ExternalSecure: false
-# NOTE: The API is now configured to run on port 8081 instead of the default 8080.
-# Ensure that port 8081 does not conflict with other services in your development environment.
-Port: 8081
-ExternalPort: 8081
```

Comment:
Is this not needed anymore?

Reply:
I moved this config to the `env.prod.test-functional-ui` file.It seems to me it makes sense to have listen and connection relevant configs together.